client/my-sites/customer-home/main.jsx

```diff
@@ -127,6 +134,14 @@ const Home = ( {
 							<Tertiary cards={ layout.tertiary } />
 						</div>
 					</div>
+					<div className="payments-features-video__modal">
+						<VideoModal
+							isVisible={ isModalOpen }
+							onClose={ closeModal }
+							onOpen={ openModal }
+							courseSlug={ COURSE_SLUGS.PAYMENTS_FEATURES }
+						/>
+					</div>
```

Comment:
I feel conflicted about adding the modal to the My Home's main script.

Since there is already another modal (`PluginsAnnouncementModal`), I wonder if it could make sense to create a new location dedicated to modals.

Something like:
```js
// main.js

import Modals from 'calypso/my-sites/customer-home/locations/modals';

<Main>
  <Primary />
  <Secondary />
  <Tertiary />
  <Modals />
</Main>
```

```js
// locations/modals/index.js

import PluginsAnnouncementModal from 'calypso/my-sites/plugins/plugins-announcement-modal';

const { isModalOpen, openModal, closeModal } = useRouteModal( /* the payment course */ );

<>
  <PluginsAnnouncementModal />
  <VideoModal /* the payment course */ />
</>
```

Reply:
Done